src/test/java/tests/BaseTest.java

```diff
@@ -28,22 +28,21 @@ public void setUpReport()   {
         extent.attachReporter(spark);
     }
 
-    @BeforeTest
+    @BeforeClass
     public void setUp()    {
         driver = WebDriverManager.chromedriver().create();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         driver.manage().window().maximize();
         driver.get(ConfProperties.getProperty("mainPage"));
     }
 
-    @AfterTest
+    @AfterClass
     public void closeDriver()  {
-        driver.close();
+        driver.quit();
     }
 
     @AfterSuite
     public void shutDown()  {
         extent.flush();
-        driver.quit();
     }
 }
```

Comment:
Changes in annotations to this class performed to rollback previous merge to main branch